app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
i actually prefer to put a space between attributes and nodes, it makes reading it easier.